diff -NaBur configure.ac configure.ac
--- configure.ac	2021-09-11 12:52:47.000000000 +0800
+++ configure.ac	2023-06-18 07:44:58.447612000 +0800
@@ -163,7 +163,7 @@ m4_define([GD_LIB_CHECK], [dnl
       dnl Merge the flags into the main LIBS/CPPFLAGS.
       $1][_LIBS="$gd_lib_ldflags $][$1][_LIBS"
       $1][_CFLAGS="$gd_lib_cflags $][$1][_CFLAGS"
-      AS_VAR_APPEND([LIBS], [" $][$1][_LIBS"])
+      LIBS=["$][$1][_LIBS $LIBS"]
       AS_VAR_APPEND([CPPFLAGS], [" $][$1][_CFLAGS"])
       if test -z "$gd_require_pkg_name"; then
         AS_VAR_APPEND([LIBS_PRIVATES], [" $][$1][_LIBS"])

